build_scripts/docker_image_creator.py

```diff
@@ -0,0 +1,59 @@
+"""
+Script to run docker image creation
+Run the command:
+    python docker_image_creator.py optimized_binary_link gpu|cpu tensorflow_version python_version
+"""
+import argparse
+import os
+import sys
+
+## GLOBALS
+# Allows user to run script anywhere
+BASE_PATH = ""
```

Comment:
Use single quotes instead of double quotes for single-line strings - I know it's arbitrary, but we try to favor using single quotes in our codebase